build/constants.yaml

```diff
@@ -107,9 +107,9 @@
     final-stretch-dot-net-60-sdk-version: 6.0.301
     final-stretch-dot-net-core-app-60-run-time-version: 6.0.6
     final-stretch-aspnet-core-app-60-run-time-version: 6.0.6
-    final-stretch-dot-net-70-sdk-version: 7.0.100
-    final-stretch-dot-net-core-app-70-run-time-version: 7.0.0
-    final-stretch-aspnet-core-app-70-run-time-version: 7.0.0
+    final-stretch-dot-net-70-sdk-version: 7.0.203
```

Comment:
is this stretch version update related to the build issue?

Reply:
That was my guess, but perhaps not the case. Looking into it

Reply:
Ok so it was to do with the incompatible versions, since some new ones were added with https://github.com/microsoft/Oryx/pull/1998.

I've merged these changes into there, so I will close this PR and we can merge in that one